app/assets/stylesheets/dsfr.scss

```diff
@@ -1,6 +1,6 @@
 @import "colors";
 // override default text underline of dsfr
-body [href] {
+body [href]:not(.fr-footer__top-link):not(.fr-footer__bottom-link):not(.fr-footer__content-link):not(.fr-footer__bottom-copy *) {
   background-image: none;
```

Comment:
On verra plus tard mais je pense qu'on devra revoir la façon dont on gère le conflit des liens entre ceux "historiques" et ceux du dsfr. Dans ma PR j'en ai aussi plusieurs à excepter et ça va vite devenir ingérable et problématique le jour où on pourra enlever le style historique. Peut-être juste avec un sélecteur css plus générique  `body[href]:not(.fr-)` pour matcher tout le dsfr.